static/js/all.js

```diff
@@ -91,7 +91,7 @@ var payFeeAmount = function() {
     payFeeElement.text('');
   } else {
     var totalAmount = (goalAmount + .30) / (1 - 0.022);
-    var feeAmount = Math.floor((totalAmount - goalAmount) * 100) / 100;
+    var feeAmount = Math.round((totalAmount - goalAmount) * 100) / 100;
```

Comment:
Small thing but for clarity we could mention that the `feeAmount` equation rounds to two decimals. That formula made me have to stop and think and a comment could maybe quickly convey what's that's doing.

Reply:
Great idea 💡 It also took me a sec to realize what this line was doing!

Reply:
Nice like to the stripe docs too! Thanks @djpeacher